### PR TITLE
Updated documentation for tenant access set command in admin-cli to keep it consistent with the output of the CLI.

### PR DESCRIPTION
**Pull request by tusharnt**
Updated documentation for tenant access set command in admin-cli to keep it consistent with the output of the CLI.

### CODE REVIEW

File: docs/user-guide/admin-cli.md [hunk 1]
```diff
@@ -390,8 +390,8 @@ usage: vmdkops_admin.py tenant access set [-h]
                                           [--volume-totalsize Num{MB,GB,TB} - e.g. 2TB]
                                           --name NAME
                                           [--volume-maxsize Num{MB,GB,TB} - e.g. 2TB]
-                                          [--allow-create] --datastore
-                                          DATASTORE
+                                          [--allow-create ALLOW_CREATE]
```
Comment (shaominchen) [inline docs/user-guide/admin-cli.md]: 1. How about changing "ALLOW_CREATE" to "True|False" to make it clearer?
2. We need to make the same change for line 300.

File: docs/user-guide/admin-cli.md [hunk 2]
```diff
@@ -401,7 +401,10 @@ optional arguments:
   --name NAME           Tenant name
   --volume-maxsize Num{MB,GB,TB} - e.g. 2TB
                         Maximum size of the volume that can be created
-  --allow-create        Allow create and delete on datastore if set to True
+  --allow-create ALLOW_CREATE
+                        Allow create and delete on datastore if set to True;
```
Comment (shaominchen) [inline docs/user-guide/admin-cli.md]: We need to make the same change for line 311.

File: docs/user-guide/admin-cli.md (hunk 1 above)
Comment (tusharnt) [inline docs/user-guide/admin-cli.md]: Done

Comment (tusharnt) [inline docs/user-guide/admin-cli.md]: As we discussed, we will address the change to make add and set commands consistent separately.

File: docs/user-guide/admin-cli.md (hunk 2 above)
Comment (tusharnt) [inline docs/user-guide/admin-cli.md]: As we discussed, we will address the change to make add and set commands consistent separately.

**Comment by tusharnt** [comment]
@pshahzeb is updating the CLI as part of issue #918 and will keep it consistent with the documentation changes done in this PR

File: docs/user-guide/admin-cli.md [hunk 3]
```diff
@@ -390,8 +390,8 @@ usage: vmdkops_admin.py tenant access set [-h]
                                           [--volume-totalsize Num{MB,GB,TB} - e.g. 2TB]
                                           --name NAME
                                           [--volume-maxsize Num{MB,GB,TB} - e.g. 2TB]
-                                          [--allow-create] --datastore
-                                          DATASTORE
+                                          [--allow-create Value{True|False} e.g. True]
```
Comment (shaominchen) [inline docs/user-guide/admin-cli.md]: Nit: I think we can remove "e.g. True"

File: docs/user-guide/admin-cli.md [hunk 4]
```diff
@@ -401,7 +401,10 @@ optional arguments:
   --name NAME           Tenant name
   --volume-maxsize Num{MB,GB,TB} - e.g. 2TB
                         Maximum size of the volume that can be created
-  --allow-create        Allow create and delete on datastore if set to True
+  --allow-create Value{True|False} - e.g. True
```
Comment (shaominchen) [inline docs/user-guide/admin-cli.md]: Same as above.

**Comment by tusharnt** [comment]
CI passed - https://ci.vmware.run/vmware/docker-volume-vsphere/1426

File: docs/user-guide/admin-cli.md (hunk 3 above)
Comment (tusharnt) [inline docs/user-guide/admin-cli.md]: I would prefer to keep it as to keep it consistent with the rest.

File: docs/user-guide/admin-cli.md (hunk 4 above)
Comment (tusharnt) [inline docs/user-guide/admin-cli.md]: I would prefer to keep it as to keep it consistent with the rest.

**Pull request merged**